crates/iceberg/src/spec/manifest.rs

```diff
@@ -819,6 +849,49 @@ impl ManifestEntry {
             ManifestStatus::Added | ManifestStatus::Existing
         )
     }
+
+    /// Content type of this manifest entry.
+    pub fn content_type(&self) -> DataContentType {
+        self.data_file.content
+    }
+
+    /// Data file path of this manifest entry.
+    pub fn file_path(&self) -> &str {
+        &self.data_file.file_path
+    }
+
+    /// Inherit data from snapshot, such as snapshot id, sequence number.
+    pub(crate) fn inherit_data(&mut self, snapshot_entry: &ManifestListEntry) {
```

Comment:
I think the naming is confusing here. Inheritance is about inheriting from the manifest-list to the manifest-entry, not from the snapshot.

Reply:
Fixed comments.